### ISSUE TITLE
Config questions

### ISSUE THREAD

**Issue by AlexKuhnle** [jviquerat/dragonfly#13]
I've tried to replicate your config now, but don't think I'm there yet. This is what it currently looks like:

```python
{
    "agent": "tensorforce",
    "policy": {"network": {"type": "auto", "size": 16, "rnn": false}, "temperature": 1.0},
    "memory": {"type": "recent"},
    "update": {"unit": "timesteps", "batch_size": 128, "frequency": 128},
    "optimizer": {"optimizer": "nadam", "learning_rate": 0.01, "multi_step": 16, "subsampling_fraction": 0.5, "clipnorm": 0.5},
    "objective": {"type": "policy_gradient", "ratio_based": true, "clipping_value": 0.1},
    "reward_estimation": {"horizon": 32, "discount": 0.99, "predict_horizon_values": "early", "estimate_advantage": true, "predict_action_values": false, "predict_terminal_values": false},
    "baseline_policy": {"network": {"type": "auto", "size": 16, "rnn": false}},
    "baseline_optimizer": {"optimizer": "nadam", "learning_rate": 0.01, "clipnorm": 0.5, "multi_step": 16},
    "baseline_objective": {"type": "value", "value": "state"},
    "preprocessing": null,
    "exploration": 0.0,
    "variable_noise": 0.0,
    "l2_regularization": 0.0,
    "entropy_regularization": 0.01
}
```

And here my learning curve:

![jonathan](https://user-images.githubusercontent.com/14937649/85330934-23caa580-b4cd-11ea-848a-4916b1b01857.png)

Clearly not there yet... :-)

May have a few more questions when having a closer look, but one thing I already came across: You're using batch normalization... does that help a lot? Would you mind trying what happens if you run it without?
